src/main/resources/io/viash/platforms/nextflow/WorkflowHelper.nf

```diff
@@ -603,10 +606,467 @@ def paramsToList(params, config) {
 }
 
 def paramsToChannel(params, config) {
+  System.err.println("Warning: paramsToChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   Channel.fromList(paramsToList(params, config))
 }
 
 def viashChannel(params, config) {
+  System.err.println("Warning: viashChannel has deprecated in Viash 0.7.0. " +
+                     "Please use a combination of channelFromParams and preprocessInputs.")
   paramsToChannel(params, config)
     | map{tup -> [tup.id, tup]}
 }
+
+/**
+ * From a viash config, gather the config entries 
+ * for arguments that have property 'multiple: true'
+ */
+
+private List<Map> _getMultiParameterSettings(Map config) {
+  def configArguments = config.functionality.allArguments
+  def plainNameArguments = configArguments.findAll{it.containsKey("plainName")}
```

Comment:
why are you filtering based on whether it contains `plainName`? I think an entry in `.allArguments` should always have a `.plainName`?

Reply:
I thought this was part of the old functionality, but it is not, I will remove the filtering